opsdroid/parsers/watson.py

```diff
@@ -0,0 +1,95 @@
+"""A helper function for parsing and executing IBM watson skills."""
+import logging
+import json
+
+from ibm_watson import AssistantV2
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+from ibm_watson import ApiException
+
+from opsdroid.const import WATSON_API_ENDPOINT
+from opsdroid.const import WATSON_API_VERSION
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def get_session_id(service, config):
+    """Authenticate and get session id from watson."""
+    service.set_service_url(
+        "https://{}{}".format(config["gateway"], WATSON_API_ENDPOINT)
```

Comment:
I would be tempted to change the `WATSON_API_ENDPOINT` in the const file to be `"https://{gateway}.watsonplatform.net/assistant/api"` and then change this line to be

```suggestion
        WATSON_API_ENDPOINT.format(gateway=config["gateway"])
```